tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -797,6 +797,26 @@ void test_float_specs() {
             [](auto ch) { return ch == charT{'0'}; }));
     }
 
+    // Test type specifiers together with precision beyond _Max_precision
+    if constexpr (is_same_v<charT, char>) {
+        charT buffer[2048];
+
+        string_view expected{buffer, to_chars(buffer, buffer + sizeof(buffer), value, chars_format::general, 2000).ptr};

```

Comment:
This should use `size` or `begin` and `end` instead of `sizeof` (ditto below). Why is it only testing `char` and not `wchar_t`?

Reply:
as far as I can tell `to_chars` only takes `char` https://en.cppreference.com/w/cpp/utility/to_chars